code-challenges.test.js

```diff
@@ -25,35 +25,146 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
-// b) Create the function that makes the test pass.
+// Psuedo:
+// Create a new test using the descrive keyword
+// designate the name of our tested function inside the '' within the test.
+// Make sure to remember parenthesis for the call back function.
+// Use the 'it' keyword to describe inside of a string what the function will do.
+// Create 3 different expect statements for the outcomes of the objects below.
+
+// describe("divBYThree", () => {
+//     it("function will return string interpelation the number is divisible by three, or the number is not divisible by three.", () => {
+//       expect(divBYThree({ number: 15 })).toBe('15 is divisible by three.')
```

Comment:
Good job on constructing your test! One thing I want you to consider is that in your expect statement you are invoking a function, so rather than using the the key:values pair as the argument what should we use as the argument to avoid getting a syntax error?